### ISSUE TITLE
Remove download zip step from grunt build

### ISSUE THREAD

**Issue by mic-smith** [ScottLogic/StockFlux#712]
No longer needed now after #709

**Comment by WPFerg** [comment]
I think for to do this well we'll need a list of download links (ie, updating/non-updating download links in the GH release). Also for running off of dev, we'd need an additional link to the CDN'd zip pointer.

**Comment by mic-smith** [comment]
install.html added to each build which redirects to the zip download link
